### DISCUSSION TITLE
Fireblocks no longer showing up in web3modal

### DISCUSSION THREAD

**Discussion by degenSumi** [WalletConnect/walletconnect-monorepo#2800] [Developer Support] [answered]
Earlier fireblocks used to show up in web3modal but now it no longer there sombody please help me i am using web3modal version 1.9.9

**Comment by JeroldG** [comment, marked as answer]
this one is now fixed, refer to V2 dapp showing FIreblocks 
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/131778446/c3661a9f-87aa-44a9-bd75-b40e1465d6ca)
